test/Parse/borrow_expr.swift

```diff
@@ -7,7 +7,7 @@ func testGlobal() {
 
 func testLet() {
     let t = String()
-    let _ = _borrow t
+    let _ = _borrow t // expected-error {{}}
 }
```

Comment:
why are we now making it an error to borrow a let?

Reply:
The reason why is that lets are always borrowed already. You don't need to apply the borrow operator to it. Users have gotten confused about this in the past (the specific confusion is... why does borrow do nothing in this case... I don't have a copy whether or not I use borrow). Better to just error on it.